csharp/Platform.Collections/IDictionaryExtensions.cs

```diff
@@ -4,17 +4,57 @@
 
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
-namespace Platform.Collections
+namespace Platform.Collections 
 {
+    /// <summary>
+    /// <para>Represents a set of extension methods <see cref="IDictionary{TKey, TValue}"/> array.</para>
+    /// <para>Представляет набор методов расширения <see cref="IDictionary{TKey, TValue}"/>.</para> 
+    /// </summary>  
     public static class IDictionaryExtensions
     {
+        /// <summary>    
+        /// <para>We get the default value.</para> 
+        /// <para>Получаем дефолтное значение.</para> 
+        /// </summary>  
+        /// <typeparam name="dictionary"> 
+        /// <para>Dictionary that is a key pair:meaning.</para> 
+        /// <para>Словарь который пара ключ:значение</para> 
+        /// </typeparam>  
+        /// <typeparam name="key"> 
+        /// <para>Key.</para> 
+        /// <para>Ключ.</para> 
+        /// </typeparam> 
+        /// <returns> 
+        /// <para>The value of the active dictionary.</para> 
+        /// <para>Значение активного словаря.</para> 
+        /// </returns>        
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static TValue GetOrDefault<TKey, TValue>(this IDictionary<TKey, TValue> dictionary, TKey key)
         {
             dictionary.TryGetValue(key, out TValue value);
             return value;
         }
 
+        /// <summary>    
+        /// <para>Adding a value.</para> 
+        /// <para>Добавляем значение.</para> 
+        /// </summary>   
+        /// <typeparam name="dictionary"> 
+        /// <para>Dictionary that is a key pair:meaning.</para> 
+        /// <para>Словарь который пара ключ:значение</para> 
+        /// </typeparam>  
+        /// <typeparam name="key"> 
+        /// <para>Key.</para> 
+        /// <para>Ключ.</para> 
+        /// </typeparam>  
+        /// <typeparam name="valueFactory">  
+        /// <para>The function, will return the value of the argument, which is the key.</para> 
+        /// <para>Функция, вернет значение по аргументу, который ключ.</para> 
+        /// </typeparam> 
+        /// <returns> 
+        /// <para>Value that is not in the dictionary.</para> 
```

Comment:
`The value associated with the <paramref name="key"/> in the <paramref name="dictionary"/> if it exists; otherwise new value.`